src/frontend/src/instance/grpc.rs

```diff
@@ -85,6 +85,7 @@ impl GrpcQueryHandler for Instance {
                             start: promql.start,
                             end: promql.end,
                             step: promql.step,
+                            lookback: DEFAULT_LOOKBACK_STRING.to_string(),
```

Comment:
@etolbakov, I wonder if we should update the gRPC definitions later to support passing lookback parameters here.

cc @waynexia @shuiyisong

Reply:
yeah, we can consider adding that.